.travis.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
And a bare document does not use the markers, which is totally valid. and in this case, using a bare document is more readable than using an explicit document